src/3d/qgswindow3dengine.cpp

```diff
@@ -35,7 +38,7 @@ QgsWindow3DEngine::QgsWindow3DEngine( Qgs3DMapCanvas *parent )
   mMapCanvas3D->setActiveFrameGraph( mFrameGraph->frameGraphRoot() );
 
   // force switching to no shadow rendering
-  setShadowRenderingEnabled( false );
+  mFrameGraph->setRenderViewEnabled( QgsFrameGraph::SHADOW_RENDERVIEW, false );
```

Comment:
`mFrameGraph->shadowRenderView()->setEnabled( false );` would be IMHO preferred over the access through some constants  (same in offscreen engine)

Reply:
This means I would have an accessor for each renderview in the framegraph?! This is not the modular approach I envisioned.

I am agree for the main renderviews as forward and shadows but the other renderviews may be (should be) accessed via constant.